.changeset/hot-falcons-grab.md

```diff
@@ -6,8 +6,9 @@
 "section": other
 ---
 
-Change when the `op` event on `IFluidDataStoreRuntimeEvents` and `IContainerRuntimeBaseEvents` is emitted
+The op event on IFluidDataStoreRuntimeEvents and IContainerRuntimeBaseEvents is emitted at a different time
 
-Previous behavior - It was emitted immediately after an op was processed and before the next op was processed.
+Previously, in versions 2.4 and below, the `op` event was emitted immediately after an op was processed and before the next op was processed.
 
-New behavior - It will still be emitted after an op is processed but it may not be immediate. Other ops in a batch may be processed before the op event is emitted for an op.
+In versions 2.5.0 and beyond, the `op` event will be emitted after an op is processed but it may not be immediate. In addition, other ops in a
```

Comment:
```suggestion
In versions 2.5.0 and beyond, the `op` event will be emitted after an op is processed, but it may not be immediate. In addition, other ops in a
```